funfact/model/_factorization.py

```diff
@@ -47,6 +50,17 @@ def forward(self):
         '''Evaluate the tensor expression the result.'''
         return self.tsrex | self._evaluator
 
+    def getelements(self, idx):
```

Comment:
It's better to name it `_get_elements` since we don't want users to call it directly.